app/javascript/components/teams/team-members.vue

```diff
@@ -1,101 +1,102 @@
 <template>
-  <div class="flex flex-col p-2 space-y-3">
-    <span class="pl-1 text-xl font-medium">Miembros</span>
-    <div class="grid grid-cols-6 grid-rows-3 gap-1">
+  <div>
+    <div class="flex-col hidden sm:flex space-y-3">
+      <span class="text-xl font-medium">{{ $t('teams.members') }}</span>
       <div
-        v-for="member in members.slice(0, 15)"
-        :key="member.id"
-        class="flex items-center justify-center w-12 h-12 overflow-hidden border-2 border-blue-500 rounded-full"
+        class="grid grid-cols-6 grid-rows-3 gap-1"
       >
         <img
+          v-for="member in members.slice(0, MAX_MEMBERS_GRID)"
+          :key="member.id"
+          class="w-12 h-12 border-2 border-blue-500 rounded-full cursor-pointer"
           :src="member.attributes.profilePictureUrl"
-          class="object-cover h-12"
+          :title="member.attributes.fullName"
         >
+        <div
+          class="flex flex-row items-center text-xs sm:justify-center space-x-1 sm:mt-1 sm:w-full sm:col-span-3"
+          v-if="members.length > MAX_MEMBERS_GRID"
+        >
+          <span
+            class="ml-1 text-sm font-medium text-blue-500 sm:ml-0"
+          >
+            +{{ members.length - MAX_MEMBERS_GRID }} {{ $t('teams.people') }}
+          </span>
+        </div>
       </div>
+      <team-join-leave-btn class="sm:flex" />
+    </div>
+
+    <div
+      class="flex flex-col p-2 shadow space-y-3 rounded-xl sm:hidden"
+    >
       <div
-        v-if="members.length > MAX_MEMBERS"
-        class="flex items-center justify-end w-full h-full text-blue-400 col-span-3"
+        class="flex flex-row items-center justify-between w-full"
       >
-        + {{ members.length - MAX_MEMBERS }} personas
+        <span class="text-base font-medium ">{{ $t('teams.members') }}</span>
+
+        <div
+          class="flex flex-row items-center text-xs"
+          v-if="!expanded"
+        >
+          <team-members-circles
+            :members="members"
+            :text-classes="'text-blue-500 font-semibold'"
+          />
+        </div>
+        <inline-svg
+          class="w-4 h-4"
+          @click="expand"
+          :src="require(`../../../assets/images/icons/${icon}`)"
```

Comment:
Ídem lo de los _assets_ acá!

```suggestion
          :src="require(`assets/images/icons/${icon}`)"
```